test/fuzz/fuzz_runner.cc

```diff
@@ -47,6 +47,10 @@ void Runner::setupEnvironment(int argc, char** argv, spdlog::level::level_enum d
   static auto* logging_context =
       new Logger::Context(log_level_, TestEnvironment::getOptions().logFormat(), *lock, false);
   UNREFERENCED_PARAMETER(logging_context);
+
+  // Suppress all libprotobuf logging as long as this object exists.
+  // For fuzzing, this disables all logs when parsing text-format corpus fails.
```

Comment:
nit: all non-fatal logs. Please add link: https://github.com/protocolbuffers/protobuf/blob/4059c61f27eb1b06c4ee979546a238be792df0a4/src/google/protobuf/stubs/logging.h#L223

Reply:
> when parsing text-format corpus fails

Or produces other parsing text-format warnings (like use of deprecated fields)

Reply:
Done